[PATCH] Set multipath to 514 and disable bgp-vnc for optimization

From: dgsudharsan <sudharsand@nvidia.com>
Date: Wed, 4 Dec 2024 20:39:46 +0000
Subject: [PATCH] Set multipath to 514 and disable bgp-vnc for optimization


---
 debian/rules |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/debian/rules b/debian/rules
index e0b626fbf..41b6c9f57 100755
--- a/debian/rules
+++ b/debian/rules
@@ -73,8 +73,7 @@ override_dh_auto_configure:
 		--disable-protobuf \
 		--disable-zeromq \
 		--enable-ospfapi \
-		--enable-bgp-vnc \
-		--enable-multipath=256 \
+		--enable-multipath=514 \
 		--enable-pcre2posix \
 		\
 		--enable-user=frr \
